bokehjs/src/coffee/source/ajax_data_source.coffee

```diff
@@ -27,7 +27,8 @@ class AjaxDataSource extends RemoteDataSource.RemoteDataSource
       xhrField :
         withCredentials : true
       method : @get('method')
-      contentType : 'application/json'
+      contentType : @get('content_type')
+      headers : @get('http_headers')
```

Comment:
or could do the server in node of course, but Python might be more familiar. node server could avoid starting a new process.